### PR TITLE
Upgrade i18next and react-i18next to latest across monorepo

### LINKED ISSUE #2308: Upgrade i18next

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

Upgrade i18next and react-i18next to latest across monorepo

### Tested

Ran wallet app, confirmed all looks good, changed language, saw change
Ditto for verifier app
Ran notification service

### Other changes

Patch small issue in run_app script where sometimes `react-native start` fails the first time

### Related issues

- Fixes #2308 

### Backwards compatibility

Yes

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -18407,23 +18382,20 @@ i18next-xhr-backend@^3.0.0:
...
-  version "12.1.0"
-  resolved "https://registry.yarnpkg.com/i18next/-/i18next-12.1.0.tgz#387bf4b94d05b0160b6a41d001a6b360e384bdb1"
-  integrity sha512-AexmwGkKxwKfo5fGeXTWEY4xqzRPigQ1S/0InOUUVziGO54cd4fKyYK8ED1Thx9fd+WA3fRSZ+1iekvFQMbsFw==
-
 i18next@^18.0.1:
   version "18.0.1"
   resolved "https://registry.yarnpkg.com/i18next/-/i18next-18.0.1.tgz#10841209f6983df4d8e367204da403d8cbc609cf"
   integrity sha512-KWd9qMFXw0qjxF7cTAqQselPCYoHfaLvBs8c6JcNzaQKVxbAlE/wv9EZXPy+JlKUcXCT0qgjcmxrJjmbnEp60A==
   dependencies:
     "@babel/runtime" "^7.3.1"
```
Comment (jeanregisser) [inline yarn.lock]: Do you know why this one is still there?

**Review by jeanregisser — APPROVED** [review]
Excellent 👍

File: yarn.lock (hunk 1 above)
Comment (jmrossy) [reply yarn.lock]: the web package, which I did not change

**Comment by jmrossy** [comment]
Looks like there's an issue with the nav headers I missed. Investigating

**Comment by jmrossy** [comment]
Need to hoist statics to head header back. Will fix tmrw

**Pull request merged**